digdag-core/src/main/java/io/digdag/core/agent/RequireOperatorFactory.java

```diff
@@ -65,43 +68,149 @@ private RequireOperator(OperatorContext context, TaskCallbackApi callback)
         public TaskResult runTask()
         {
             Config config = request.getConfig();
+            Config localConfig = request.getLocalConfig();
+            Config lastStateParams = request.getLastStateParams();
+
             String workflowName = config.get("_command", String.class);
-            int projectId = config.get("project_id", int.class);
             Instant instant = config.get("session_time", Instant.class);
             boolean ignoreFailure = config.get("ignore_failure", boolean.class, false);
-            Optional<String> retryAttemptName = config.getOptional("retry_attempt_name", String.class);
+            OptionRerunOn rerunOn = OptionRerunOn.of(config.get("rerun_on", String.class, "none"));
+
+            //retry_attempt_name is set from local config only.
+            Optional<String> retryAttemptName = localConfig.getOptional("retry_attempt_name", String.class);
```

Comment:
I'm wondering if this scope reduction of `retry_attempt_name` can affect existing workflows...

Reply:
Yes. There is possibility as discussion in person.
Let me discuss again in a meeting.

Reply:
Just noticed you've already mentioned it in the document :)